picaso/justdoit.py

```diff
@@ -2671,12 +2673,12 @@ def atmosphere_4d(self, ds=None, shift=None, plot=True, iz_plot=0,verbose=True,
         if isinstance(shift, type(None)):
             shift = np.zeros(len(phases))
         
-        if zero_point == 'night_transit':
+        if zero_point == 'night_transit':   ## does not work for reflected case!
```

Comment:
So that is why there is now the "reorder" option in the plotting. Do you recommend I just take this out?

Reply:
Yes I think it'd be best to remove this since reorder_output will work for both the reflected and thermal case. If someone were to use zero_point='night_transit' for the reflected case, they would get bad output from atmosphere_4d and clouds_4d and it wouldn't be immediately clear that the output is wrong.